docs/docs/concepts/blueprints.md

```diff
@@ -1,61 +1,153 @@
+| extends          | string                            | A reference to another blueprint. The blueprint will inherit attributes from the referenced blueprint. |
+| attributes       | `system/SIMOS/BlueprintAttribute` | List of attributes.                                                                                    | 
+
+:::info
+
+An attribute that can be complex (reference to another blueprint) or primitive (string, integer, number, boolean). 
+
+:::
+
+### system/SIMOS/BlueprintAttribute
+
+Has the following attributes, where the `attributeType` and `type` is required.  
+
+| Attribute     | Type     | Description                                                                                                                                                                                                |
+|---------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| attributeType | string   | The attribute type. Can be a reference to another blueprint, or a primitive type (string, integer, number, boolean).                                                                                       |
+| name          | string   | Name of the attribute.                                                                                                                                                                                     |
+| contained     | boolean  | Describes the conceptual belonging of one entity to another. In other words, if an entity must always be considered in the context of it's "parent" (true), or the entity has meaning on it's own (false). |
+| dimensions    | string   | Degree of dimensions. Format; A list of integers, each integer defines the size of a dimension. "*"(wildcard) has the special meaning of "unfixed size                                                     |
+| optional      | boolean  | If the attribute is optional or required                                                                                                                                                                   |
+| default       | any json | Default value for the attribute. Same type as the attribute itself                                                                                                                                         |
+| enumType      | string   | A reference to an Enum entity. Value should hold the chosen enum value in the entity                                                                                                                       |
+
+## References 
+
+It is possible to reference other blueprint from the `type`, `extends`, `attributeType`, and `_blueprintPath_` attributes.
```

Comment:
I suppose you can only reference one blueprint? Or can you for instance extend multiple blueprints?
```suggestion
It is possible to reference another blueprint from the `type`, `extends`, `attributeType`, and `_blueprintPath_` attributes.
```

Reply:
Extends is an array of references [reference1, reference2, ...]. So it's possible to extend from multiple blueprints.